apps/src/codebridge/Workspace/HeaderButtons.tsx

```diff
@@ -1,15 +1,50 @@
 import React from 'react';
 
+import {Button} from '@cdo/apps/componentLibrary/button';
+import {TooltipProps, WithTooltip} from '@cdo/apps/componentLibrary/tooltip';
 import VersionHistoryButton from '@cdo/apps/lab2/views/components/versionHistory/VersionHistoryButton';
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
+import commonI18n from '@cdo/locale';
 
 import {useCodebridgeContext} from '../codebridgeContext';
 
+import moduleStyles from './workspace.module.scss';
+import darkModeStyles from '@cdo/apps/lab2/styles/dark-mode.module.scss';
+
 const WorkspaceHeaderButtons: React.FunctionComponent = () => {
   const {startSource} = useCodebridgeContext();
 
+  const appName = useAppSelector(state => state.lab.levelProperties?.appName);
+
+  const feedbackTooltipProps: TooltipProps = {
+    text: commonI18n.feedback(),
+    direction: 'onLeft',
+    tooltipId: 'feedback-tooltip',
+    size: 'xs',
+    className: darkModeStyles.tooltipLeft,
+  };
+
+  const openFeedbackForm = () => {
+    window.open('https://forms.gle/Z4FsGMFzE4NrFp369', '_blank');
+  };
+
   return (
-    <div>
+    <div className={moduleStyles.rightHeaderButtons}>
       <VersionHistoryButton startSource={startSource} />
+      {appName === 'pythonlab' && (
```

Comment:
Do we have a front-end source of truth for app names? We've got checks for `'pythonlab'` in half a dozen files, and I wonder if it's worth having a constant. It doesn't need to block this PR; it's just a thought/question.

Reply:
kind of; we have a type [here](https://github.com/code-dot-org/code-dot-org/blob/b369866c6e90a061bbe53e19736dc0fcaaf087dc/apps/src/lab2/types.ts#L279) which is built off of this [object](https://github.com/code-dot-org/code-dot-org/blob/b369866c6e90a061bbe53e19736dc0fcaaf087dc/apps/lab2EntryPoints.ts#L19) but I don't think we can extract a constant from it. We probably should have a constant though; other lab2 labs have the same problem